src/app/auth/auth.module.ts

```diff
@@ -0,0 +1,29 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+
+import { SharedModule } from '@app/shared/shared.module';
+
+import {
+  LoginComponent,
+  LogoutComponent
+} from '@app/auth/pages';
+
+import { authRoutes } from './auth.routes';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    SharedModule,
+    // Routing
+    RouterModule.forRoot(
```

Comment:
When you import `RouterModule` in the nested module, you should use method `.forChild`, because it methods doesn't include router service ( to not duplicate it ). For more information you may read this article: https://angular.io/api/router/RouterModule#usage-notes

Reply:
fixed